### ISSUE TITLE
As a user I can view a map of all the events and discord records

### ISSUE THREAD

**Issue by Seggellion** [Seggellion/bar-citizen-web#8]

**Comment by ShaneAusTx** [comment]
On website there should be a button for "Find a Bar Citizen Event" (which would show events and hide Discords). And, a button for "Find a Community Discord" (which would show Discords and hide Events).

**Comment by Seggellion** [comment]
@ShaneAusTx  yes these buttons have been added, they're just at the bottom and broken:



![Image](https://github.com/Seggellion/bar-citizen-web/assets/30736834/e7b0f1fe-6e55-492f-8dd2-e13bd5d60e3f)

I was hoping to get to that tonight but I just didn't have enough time. I'll try again tomorrow.

**Comment by Seggellion** [comment]
I may revisit the "View Discords By Regions" view. !remindme 15 days

**Comment by ShaneAusTx** [comment]
The view discords by region would be a last item thing! However, some questions/issues:
1)  on both mobile and on desktop the icon for the Bar Citizen event is so small I can't make out what it is.  Did you use a scalable graphic?  I can make a custom one that has the shape and is an SVG - but, it needs to be a bit bigger when zoomed in.
2) When I click on the graphic I should be able to be shown the info (even if I'm not logged into the website) and be able to veiw the link to the location, the discord server link for the event, and the contact info/email for the event registrar.

#2 is most likely on the todo list already - but, I mentioned it here just to make sure.  Same for the discord links when a new user searches for a discord server.

Regarding the regional discords - (not the individual city ones) - we need to discuss how they will be shown on the discord map. I'll add that to my todo list. My thoughts are that they would not be shown on the regular/Hero page map.  They would be listed in an area under the Bar Citizen Coordinator's Resources section of the website as thier purpose it to provide assistance to regoinal coordinators.
